pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -299,8 +299,8 @@ impl<T: Config> Pallet<T> {
 				let len_fee = Self::length_to_fee(len);
 				let base_fee = Self::weight_to_fee(CAPACITY_EXTRINSIC_BASE_WEIGHT);
 
-				let adjusted_weight_fee =
-					base_fee.saturating_add(weight_fee).saturating_add(len_fee);
+				let adjusted_weight_fee = weight_fee;
```

Comment:
you didn't want to just use weight_fee below?

Reply:
yeah, I guess I could.